protocols/gossipsub/src/lib.rs

```diff
@@ -156,18 +156,53 @@ mod types;
 
 mod rpc_proto;
 
-pub use self::behaviour::{Gossipsub, GossipsubEvent, MessageAuthenticity};
```

Comment:
Deprecation type aliases for `Gossipsub` and `GossipsubEvent` are missing.

Reply:
:heavy_check_mark: